examples/simple/ads.js

```diff
@@ -25,6 +25,9 @@ var options = {
       'vid=short_onecue&correlator='
 };
 
+player.on('ads-request', function(event, data) {
+  console.log('ads-request:' + event.type + " " + data);
+});
```

Comment:
Can we move this to advanced example? Simple should be the bare minimum needed for an implementation.

Reply:
Oh, I wasn't going to actually use the fired event in the samples, this testing code got added by mistake.